src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -335,12 +335,18 @@ namespace OpenRCT2::Platform
 
     std::vector<std::string_view> GetSearchablePathsRCT1()
     {
-        return {};
+        return {
+            // game-data-packager uses this path when installing game files
+            R"(/usr/share/games/roller-coaster-tycoon)",
```

Comment:
Why are these paths wrapped in parentheses?

Reply:
Probably because they are in Platform.Win32.cpp.

But these strings don’t need to be raw anyway, so you can dispense with both the `R` and the parentheses.